### DISCUSSION TITLE
Type error when using slidev

### DISCUSSION THREAD

**Discussion by NarHakobyan** [slidevjs/slidev#1953] [Q&A] [answered]
Hey everyone, how can I use slidev autocomplete?

P.s It's running fine; just editor issue

<img width="390" alt="image" src="https://github.com/user-attachments/assets/b860e927-f421-4597-9853-062f57bab050">

**Comment by kermanx** [comment, marked as answer]
A possible solution is to use the composable: https://sli.dev/guide/global-context#composable-usage

**Comment by NarHakobyan** [comment]
Amazing, Thanks

Best Regards,
Narek
